devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1117,7 +1117,7 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
 
     if( !prop.check("HW_USE_MAS_IMU") )
     {
-        useMasIMU = false;
+        yError() << "wholeBodyDynamics : missing required group parameter HW_USE_MAS_IMU";
```

Comment:
Should it ``return false`` as well?

Reply:
Yes thanks, done in https://github.com/robotology/whole-body-estimators/pull/181/commits/3d5ff94d8cd63759c7d9657a974fec1f0495db26 .